overcloud.yml

```diff
@@ -26,7 +26,7 @@
         template:
           src: "firstboot-nvme.yaml.j2"
           dest: "/home/stack/firstboot-nvme.yaml"
-        when: nvme is defined
+        when: nvme is defined or mount_nvme is defined
```

Comment:
The template also have some fields like ComputeKernelArgs that are only needed for the passthrough case, so we should have a condition within the template.